cmd/erasure-healing_test.go

```diff
@@ -190,23 +188,24 @@ func TestHealingDanglingObject(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	for _, fsDir := range fsDirs[:4] {
-		if err = os.Chmod(fsDir, 0400); err != nil {
-			t.Fatal(err)
+	setDisks := func(newDisks ...StorageAPI) {
+		objLayer.(*erasureServerPools).serverPools[0].erasureDisksMu.Lock()
+		for i := range newDisks[:4] {
```

Comment:
minor nit: the caller can pass the appropriate sub-slice. This function can use `newDisks` as supplied.
```suggestion
		for i := range newDisks {
```